src/server/common/helpers/scan-result-response.js

```diff
@@ -21,6 +21,7 @@ function toFilesResponse(uploadId, files) {
     fileStatus: file.fileStatus,
     contentType: file.contentType,
     contentLength: file.contentLength,
+    checksumSha256: file.checksumSha256,
```

Comment:
maybe go for shorter name like just `sha256`?  The checksum prefix feels superfluous

Reply:
Yeah maybe, but also for some sha256 might not mean much?

Reply:
The other was just checksum, but then they would have to read the API to see if MD5, SHA1, SHA256, CRC32 etc. But it is at least shorter

Reply:
After all we do not specify (by name) if the contentLength is bytes, Kb or Mb either.

Reply:
:shrug:

Reply:
Settled on keeping it as `checksumSha256`